### DISCUSSION TITLE
How to I change the highlighting for builtin python classes?

### DISCUSSION THREAD

**Discussion by sophiiss** [microsoft/pylance-release#1249] [Q&A]
The highlighting for the python builtins classes (like range) is the for my own classes. Is there a way to make the builtins a different color?

**Comment by jakebailey** [comment]
We apply a "builtin" modifier to symbols that come from the builtins; you should be able to configure VS Code to have a custom color for tokens with that modifier. Something like: https://github.com/microsoft/vscode/issues/66729#issuecomment-698518458 (but matching the modifier).
